bakerydemo/base/blocks.py

```diff
@@ -42,7 +42,7 @@ class BlockQuote(StructBlock):
     """
     text = TextBlock()
     attribute_name = CharBlock(
-        blank=True, required=False, label='e.g. Guy Picciotto')
+        blank=True, required=False, label='e.g. Mary Berry')
 
```

Comment:
As much as I love Fugazi I'm not sure Guy Picciotto is appropriate as an attribute name example. Have changed to Mary Berry who's a celebrity baker here in the UK